.gitignore

```diff
@@ -1,3 +1,2 @@
 .idea
-composer.lock
```

Comment:
I see your point, do you have a policy on when to run `composer update`? Should be done on a regular basis to catch both cases.